Queue/Storage/Context.ts

```diff
@@ -2,45 +2,59 @@ import { gracely } from "gracely"
 import * as isoly from "isoly"
 import * as cloudflare from "@cloudflare/workers-types"
 import { http } from "cloudly-http"
+import { storage } from "cloudly-storage"
+import { Types } from "../../Types"
 
 export class Context {
-	private constructor(
-		private readonly state: cloudflare.DurableObjectState,
-		private readonly maxRetries = 5,
-		private readonly timeFactor = 1
-	) {}
-	async enqueue(request: http.Request.Like): Promise<void> {
-		this.tryEnqueue(request)
+	readonly alarm = new storage.DurableObject.Alarm(this.state.storage)
+	private constructor(private readonly state: cloudflare.DurableObjectState) {}
+	async enqueue(event: Types.EventBase): Promise<void> {
+		const request: http.Request.Like = {
+			method: "POST",
+			url: event.url,
+			body: { hook: event.hook, event: event.body },
+			header: { ...event.header, contentType: "application/json" },
```

Comment:
we will be sending "application/jwt", so maybe have event.header be able to override contentType.